kasa/protocol.py

```diff
@@ -14,13 +14,20 @@
 import errno
 import logging
 import struct
+import sys
 from pprint import pformat as pf
 from typing import Dict, Generator, Optional, Union
 
 from .exceptions import SmartDeviceException
 from .json import dumps as json_dumps
 from .json import loads as json_loads
 
+if sys.version_info[:2] < (3, 11):
+    from async_timeout import timeout as asyncio_timeout
+else:
+    from asyncio import timeout as asyncio_timeout
```

Comment:
Maybe keep it simple and just use async_timeout always? And maybe adding a comment for context, that the dependency can be dropped after 3.12 is the oldest supported release.

Reply:
6b4d70f6e91b1e899ae979e3415092312ae33d08